net/matrix-structs/Portfile

```diff
@@ -1,23 +0,0 @@
-# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
-
-PortSystem          1.0
-PortGroup           github 1.0
-PortGroup           cmake 1.1
-
-github.setup        mujx matrix-structs 8de04af
```

Comment:
Include `Closes: https://trac.macports.org/ticket/61139` in this commit message.

matrix-structs was merged with mtxclient, so the matrix-structs portfile should remain for one year, marked as obsolete (include the obsolete 1.0 portgroup) and `replaced_by mtxclient`. You might set the obsolete port's homepage to ​https://github.com/Nheko-Reborn/mtxclient.